### ISSUE TITLE
ReferenceError: marked is not defined

### ISSUE THREAD

**Issue by kazuki** [vanillawc/wc-markdown#12]
In v1.5.13/v1.6.1, I found ReferenceError. But my code work fine in v1.5.12.

```
Uncaught (in promise) ReferenceError: marked is not defined
    node_modules vanillawc/wc-markdown/index.js?:1729
    node_modules vanillawc/wc-markdown/index.js?:1716
    node_modules vanillawc/wc-markdown/index.js?:1701
    __insert webpack:///./node_modules/lit-html/lib/parts.js?:223
    __commitNode webpack:///./node_modules/lit-html/lib/parts.js?:230
    __commitTemplateResult webpack:///./node_modules/lit-html/lib/parts.js?:265
    commit webpack:///./node_modules/lit-html/lib/parts.js?:205
    render webpack:///./node_modules/lit-html/lib/render.js?:50
    render webpack:///./node_modules/lit-html/lib/shady-render.js?:266
    update webpack:///./node_modules/lit-element/lit-element.js?:205
    performUpdate webpack:///./node_modules/lit-element/lib/updating-element.js?:555
    _enqueueUpdate webpack:///./node_modules/lit-element/lib/updating-element.js?:514
```

`node_modules vanillawc/wc-markdown/index.js?:1729` is

```
  static toHtml(markdown) {
    return marked(markdown); << here
  }
```

Is this patch fix this issue ?

```
  static toHtml(markdown) {
    return marked2.default(markdown);
  }
```

**Comment by evanplaice** [comment]
Thank you for reporting on this. I'm working on it.

It was caused by switching the build process over to ESBuild in 1.6+. 1.7 will revert the build process back to the older setup.

---

If you want the specifics. ESBuild isn't detecting a name collision that doesn't exist. It sees the import definition and the call to the import as 2 variable declarations when one is the declaration and the second is the call to that variable.

Bundling IIFEs (ie marked) is not an ideal option but no feature-complete markdown parser written in ESM exists yet.

**Comment by evanplaice** [comment]
Scratch my previous comment. Marked now ships an ESM build in the latest. Updating and using the ESM import fixed the issue.

**Comment by evanplaice** [comment]
Fixed in [v1.6.2](https://github.com/vanillawc/wc-markdown/releases/tag/v1.6.2)
